google/cloud/pubsublite/CMakeLists.txt

```diff
@@ -123,7 +125,7 @@ target_compile_options(google_cloud_cpp_pubsublite_mocks
                        INTERFACE ${GOOGLE_CLOUD_CPP_EXCEPTIONS_FLAG})
 
 file(
-    GLOB test_files
```

Comment:
I think we should stop using globs in the CMake file too.

Reply:
Yeah, only doing half of the change didn't make sense. Fixed.